grakn-graph/src/main/java/ai/grakn/graph/internal/AbstractGraknGraph.java

```diff
@@ -774,9 +770,9 @@ public void commit() throws InvalidGraphException{
         return logs;
     }
 
-    private void closeTransaction(String closedReason){
+    private void closeTransaction(String closedReason) {
         try {
-            // TODO: We check `isOpen` because of a Titan bug which decrements the transaction counter even if the transaction is closed
+            // TODO: We check `isOpen` because of a Janus bug which decrements the transaction counter even if the transaction is closed
```

Comment:
Yes because it is a feature I patched into Janus/Titan which I have not revisited yet.